sentry/src/main/java/io/sentry/util/HintUtils.java

```diff
@@ -16,6 +20,18 @@ public final class HintUtils {
 
   private HintUtils() {}
 
+  public static void setIsFromHybridSdk(final @NotNull Hint hint, final @NotNull String sdkName) {
```

Comment:
`l`: should these 2 new methods be also marked as `@Internal`?

Reply:
The class is already internal.

Reply:
then we should probably remove it from the other methods to keep it aligned